README.md

```diff
@@ -23,26 +23,26 @@ HTTP API
 Installation
 ------------
 
-The best way to run Puppet is with [Puppet Enterprise](https://puppet.com/puppet/puppet-enterprise),
+The best way to run Puppet is with [Puppet Enterprise (PE)](https://puppet.com/puppet/puppet-enterprise),
 which also includes orchestration features, a web console, and professional support.
-[The PE documentation is available here.](https://puppet.com/docs/pe/latest)
+The PE documentation is [available here.](https://puppet.com/docs/pe/latest)
 
 To install an open source release of Puppet,
-[see the installation guide on the docs site.](http://puppet.com/docs/puppet/5.4/install_pre.html)
+[see the installation guide on the docs site.](http://puppet.com/docs/puppet/latest/install_pre.html)
 
 If you need to run Puppet from source as a tester or developer,
-[see the running from source guide on the docs site.](https://docs.puppet.com/puppet/3.8/from_source.html)
+see the [Running Puppet from Source](https://docs.puppet.com/puppet/3.8/from_source.html) guide on the docs site.
```

Comment:
@gguillotte @jtappa @Magisus Do any of you know if there is a newer version of the  running puppet from source doc that we should be pointing at here?

Reply:
There is not, there is already a ticket (PUP-8200) for this though.

Reply:
Ok